### PR TITLE
Make core work with our testnet services

### PR DESCRIPTION
**Pull request by victorkirov**
# 🔘 PR Type

What kind of change does this PR introduce?

- [x] Enhancement

# 📜 Background

Currently we don't call the correct services if the wallet is in testnet mode. This adds the functionality to route traffic to the correct services.

Issue Link: [ENG-3047](https://linear.app/xverseapp/issue/ENG-3047/make-core-work-with-our-testnet-services)
Context Link (if applicable):

# 🔄 Changes

Does this PR introduce a breaking change?

- [x] Yes, Incompatible API changes
- [ ] No, Adds functionality (backwards compatible)
- [ ] No, Bug fixes (backwards compatible)

Changes:

- The network arg has been added to all the functions that needed it

Impact:

- This allows us to use testnet

# 🖼 Screenshot / 📹 Video

# ✅ Review checklist

Please ensure the following are true before merging:

- [ ] Code Style is consistent with the project guidelines.
- [ ] Code is readable and well-commented.
- [ ] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [ ] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [ ] Code does not introduce new technical debt or issues.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -39,7 +39,6 @@ module.exports = {
           {
             loader: require.resolve('ts-loader'),
             options: {
-              transpileOnly: true,
```
Comment (teebszet) [inline webpack.config.js:42]: should we also run `tsc` on the npm lint for CI?

**Review by teebszet — APPROVED** [review]
lgtm. thanks for this!

File: api/helper.ts [hunk 2]
```diff
@@ -391,7 +391,3 @@ export function getFetchableUrl(uri: string, protocol: string): string | null {
   }
   return null;
 }
-
-export function getOrdinalImageUrl(content: string): string | null {
-  return getFetchableUrl(`${ORDINALS_URL}${content}`, 'http');
-}
```
[the remark this answers is not in the document]
Comment (victorkirov) [reply api/helper.ts:397]: Nope. It was also broken because ORDINALS_URL was actually a function 😅

File: webpack.config.js (hunk 1 above)
Comment (victorkirov) [reply webpack.config.js:42]: `tsc` is run in the build step of the CI (`build:esm` script), so that would fail in CI if there were issues 👍

**Review by teebszet — APPROVED** [review]

**Pull request merged**